Source/WebKit/Shared/WebEventModifier.h

```diff
@@ -41,6 +42,7 @@ enum class WebEventModifier : uint8_t {
     CapsLockKey = 1 << 4,
 };
 
+OptionSet<WebEventModifier> toWeb(OptionSet<WebCore::PlatformEventModifier>);
```

Comment:
I would probably call this `modifiersFromPlatformEventModifiers`, since we don't really use `toWeb` elsewhere. 

`from` would match `fromCorePage`/`Frame`/`Overlay` etc.

Reply:
I like the naming idea. Will follow suit

Reply:
Renamed in https://github.com/WebKit/WebKit/commit/1b59364ae200544534b097906f0eca943e445b7a